src/components/timeout-dialog/_timeout-dialog.scss

```diff
@@ -1,7 +1,7 @@
-@import "node_modules/govuk-frontend/govuk/settings/all";
-@import "node_modules/govuk-frontend/govuk/tools/all";
-@import "node_modules/govuk-frontend/govuk/helpers/all";
-@import "node_modules/govuk-frontend/govuk/vendor/sass-mq";
+@import "../../../../govuk-frontend/govuk/settings/all";
+@import "../../../../govuk-frontend/govuk/tools/all";
+@import "../../../../govuk-frontend/govuk/helpers/all";
+@import "../../../../govuk-frontend/govuk/vendor/sass-mq";
```

Comment:
If we want to enforce this approach we can remove `node_modules` from the import list here https://github.com/hmrc/hmrc-frontend/blob/master/tasks/gulp/compile-assets.js#L56 and people using the old approach will lead to failed builds.

Reply:
Done